.gitignore

```diff
@@ -89,3 +89,6 @@ gradlew.*
 # Ignore logs and core dumps
 *.log
 core.*
+
+# Ignore Doxygen documentation
+*/*/*/*/*/doc
```

Comment:
This should really be moved into a `.gitignore` inside of the `Decompiler` module.